docs/modules/ROOT/pages/references/config-reference.adoc

```diff
@@ -0,0 +1,43 @@
+= Configuration Reference
+
+The Operator can be configured in two ways:
+
+. Per namespace backups. Optimal for shared clusters.
+. Global settings with namespaced schedules. Optimal for private clusters.
+
+== Environment Variables
+
+`BACKUP_ANNOTATION`:: the annotation to be used for filtering, by default: `k8up.syn.tools/backup`
+`BACKUP_BACKUPCOMMANDANNOTATION`:: set the annotation name that identify the backup commands on Pods, by default: `k8up.syn.tools/backupcommand`
+`BACKUP_CHECKSCHEDULE`:: the default check schedule, by default: `0 0 * * 0`
+`BACKUP_DATAPATH`:: to which path the PVCs should get mounted in the backup container, by default: `/data`
+`BACKUP_FILEEXTENSIONANNOTATION`:: set the annotation name where the file extension is stored for backupcommands, default `k8up.syn.tools/file-extension`
+`BACKUP_GLOBALACCESSKEYID`:: set the S3 access key id to be used globaly
+`BACKUP_GLOBALKEEPJOBS`:: set the number of old backup jobs to keep globally
+`BACKUP_GLOBALREPOPASSWORD`:: set the restic repository password to be used globaly
+`BACKUP_GLOBALRESTORES3ACCESKEYID`:: set the global restore S3 accessKeyID for restores
+`BACKUP_GLOBALRESTORES3BUCKET`:: set the global restore S3 bucket for restores
+`BACKUP_GLOBALRESTORES3ENDPOINT`:: set the global restore S3 endpoint for the restores (needs the scheme [http/https]
+`BACKUP_GLOBALRESTORES3SECRETACCESSKEY`:: set the global restore S3 SecretAccessKey for restores
+`BACKUP_GLOBALS3BUCKET`:: set the S3 bucket to be used globally
+`BACKUP_GLOBALS3ENDPOINT`:: set the S3 endpoint to be used globally
+`BACKUP_GLOBALSECRETACCESSKEY`:: set the S3 secret access key to be used globaly
+`BACKUP_GLOBALSTATSURL`:: set the URL of wrestic to post additional metrics gloablly, default `""`
+`BACKUP_IMAGE`:: URL of the restic image, default: `172.30.1.1:5000/myproject/restic`
+`BACKUP_JOBNAME`:: names for the backup job objects in OpenShift, default: `backupjob`
+`BACKUP_METRICBIND`:: set the bind address for the prometheus endpoint, default: `:8080`
```

Comment:
I would love to see these environment variables being generated from code... this is a pain to maintain. Dunno how's it gonna look in K8up v2 though.
```suggestion
`BACKUP_METRICS_BINDADDRESS`:: set the bind address for the prometheus endpoint, default: `:8080`
```
 Also, some new variables are missing: 

- GlobalConcurrentArchiveJobsLimit
- GlobalConcurrentBackupJobsLimit
- GlobalConcurrentCheckJobsLimit
- GlobalConcurrentPruneJobsLimit
- GlobalConcurrentRestoreJobsLimit
- GlobalCPUResourceRequest
- GlobalCPUResourceLimit
- GlobalMemoryResourceRequest
- GlobalMemoryResourceLimit